tests/service/PatternMatchCompilationTests.fs

```diff
@@ -439,9 +439,7 @@ match Unchecked.defaultof<System.ValueType> with
 | g -> ()
 """
     assertHasSymbolUsages ["a"; "b"; "c"; "d"; "e"; "f"; "g"] checkResults
-    dumpErrors checkResults |> shouldEqual [
-        "(4,2--4,85): This rule will never be matched"
-    ]
+    dumpErrors checkResults |> shouldEqual []
```

Comment:
This is actually a regression since it now does not detect duplicated type tests.

Reply:
Yes, I need to think about that.

I think we need a column-based multi-type test construct in the TypedTree

Reply:
This has been resolved